apps/frontend/src/libs/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import { Icon } from "~/libs/components/components.js";
+import { useLocation } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type BreadcrumbItem = {
+	link?: string;
+	name: string;
+};
+
+type Properties = {
+	items?: BreadcrumbItem[];
+};
+
+const FIRST_ITEM_INDEX = 0;
+const SINGLE_ITEM_OFFSET = 1;
+
+const formatBreadcrumbName = (name: string): string => {
+	return (
+		name
+			.replaceAll("-", " ")
+			// eslint-disable-next-line unicorn/prefer-string-replace-all
+			.replace(/\b\w/g, (char) => char.toUpperCase())
+	);
+};
+
+const Breadcrumbs = ({ items }: Properties): JSX.Element => {
+	const location = useLocation();
+	const pathnames = location.pathname.split("/").filter(Boolean);
+
+	const generatedItems = pathnames.map((_, index) => {
+		const currentPathname = pathnames[index];
+		const link = `/${pathnames.slice(FIRST_ITEM_INDEX, index + SINGLE_ITEM_OFFSET).join("/")}`;
+
+		return {
+			link,
+			name: currentPathname ? formatBreadcrumbName(currentPathname) : "",
+		};
+	});
+
+	const breadcrumbs = items || generatedItems;
+
+	return (
+		<nav aria-label="breadcrumb" className={styles["breadcrumb-container"]}>
+			<ol className={styles["breadcrumb-list"]}>
+				{breadcrumbs.map((item, index) => {
+					const isLastItem = index === breadcrumbs.length - SINGLE_ITEM_OFFSET;
+
+					return (
+						<li className={styles["breadcrumb-item"]} key={index}>
+							{isLastItem ? (
+								<span className={styles["breadcrumb-current"]}>
+									{item.name}
+								</span>
+							) : (
+								<a className={styles["breadcrumb-link"]} href={item.link}>
+									{item.name}
+								</a>
+							)}
+							{!isLastItem && (
+								<span className={styles["breadcrumb-separator"]}>
+									<Icon
+										color="var(--color-border-primary)"
+										height={20}
+										name="lessThan"
+										width={20}
+									/>
```

Comment:
[PR](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/51) changed `Icon` a bit. In order to set the color of icon, you should set the color in styles of icon wrapper (`"breadcrumb-separator"` in your case). So in `styles.module.css` should be such a line
```css
.breadcrumb-separator {
  ...
  color: var(--color-border-primary);
  ...
}
```

Also replace `20` with a constant (for instance `const ICON_SIZE = 20`)

Reply:
I was told that we should include the borders for all SVG Icons we export out of Figma designs

Reply:
You mean that exported svg dimensions should be 20x20 (or other square-size according to figma design)? 
If yes, then please re-export all arrow icons, cause I didn't know that and I exported them incorrectly (without border included)